### DISCUSSION TITLE
Third party plugin/extensions system

### DISCUSSION THREAD

**Discussion by jsyrjala** [zed-industries/zed#6538] [Feature Requests]
What kind of plans or ideas Zed has for third party plugin/extensions system?

**Comment by TerminalFi** [comment]
> What kind of plans or ideas Zed has for third party plugin/extensions system?

They are considering exposing APIs through WASM but the last I understood is that they will be picky about what APIs are created. They learned a lot about API exposure when working on Atom and want to prevent over exposing APIs with Zed

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
